docs/conf.py

```diff
@@ -56,6 +59,13 @@
 myst_heading_anchors = 4
 nb_execution_mode = "off"
 
+# -- Custom Lexers -----------------------------------------------------------
+
+from lexers import FijiGroovyLexer
+
+def setup(app):
+    app.add_lexer('fijigroovy', FijiGroovyLexer)
```

Comment:
And similarly, here I would name the lexer `scijava-groovy`. With dash for clarity, assuming dashes are allowed. Or else `scijava_groovy` or `sciJavaGroovy` if dashes cannot be used.

Reply:
We can use dashes. The SciJava Groovy lexer alias is now `scijava-groovy`.